[DOCS] Remove examples of specifying a test_backends list of dicts (#7816)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations.md

	* `in` contains exactly the parameters that you want to pass in to the Expectation. `"in": {"column": "x", "min_value": 4, "strict_min": True}` in the example above is equivalent to `expect_column_max_to_be_between_custom(column="x", min_value=4, strict_min=True)`
	* `out` is based on the <TechnicalTag tag="validation_result" text="Validation Result" /> returned when executing the Expectation.
	* `exact_match_out`: if you set `exact_match_out=False`, then you don’t need to include all the elements of the Validation Result object - only the ones that are important to test.

> * `only_for` (optional): list of backends that the Expectation should work with

Reviewer: ```suggestion
* `only_for` (optional): the backends that the Expectation should use for testing
```

> * `suppress_test_for` (optional): list of backends that the Expectation does not work with

Reviewer: ```suggestion
* `suppress_test_for` (optional): the list of backends that the Expectation should not use for testing
```

> * `only_for` and `suppres_test_for` may be specified at the top-level (next to `data` and `tests`) or within specific tests (next to `title`, etc)

Reviewer: ```suggestion
* `only_for` and `suppres_test_for` can be specified at the top-level (next to `data` and `tests`) or within specific tests (next to `title`, and so on)
```
![image](https://user-images.githubusercontent.com/104205257/236830891-c34c346e-21f8-4d57-87fb-b3819d757c7a.png)
